OctoKit/Issue.swift

```diff
@@ -226,6 +227,29 @@ public extension Octokit {
             }
         }
     }
+
+    /// Fetches all comments for an issue
+    /// - Parameters:
+    /// - session: RequestKitURLSession, defaults to URLSession.sharedSession()
+    /// - owner: The user or organization that owns the repository.
+    /// - repository: The name of the repository.
+    /// - number: The number of the issue.
+    /// - page: Current page for comments pagination. `1` by default.
+    /// - perPage: Number of comments per page. `100` by default.
+    /// - completion: Callback for the outcome of the fetch.
+    @discardableResult
+    func commentsIssue(_ session: RequestKitURLSession = URLSession.shared, owner: String, repository: String, number: Int, page: String = "1", perPage: String = "100", completion: @escaping (_ response: Response<[Comment]>) -> Void) -> URLSessionDataTaskProtocol? {
```

Comment:
Could you rename this to `issueComments` as it's the comments of an issue not the issue of a list of comments, if that makes sense.
```suggestion
    func issueComments(_ session: RequestKitURLSession = URLSession.shared, owner: String, repository: String, number: Int, page: String = "1", perPage: String = "100", completion: @escaping (_ response: Response<[Comment]>) -> Void) -> URLSessionDataTaskProtocol? {
```